fusor-ember-cli/app/components/pf-modal-base.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+    openCloseModal: Ember.observer('openModal', function() {
+        if (this.get('openModal')) {
+            return Ember.$('#'+this.get('idModal')).modal('show');
```

Comment:
@cfchase, did you want to look into this more to see if we don't need to pass in `idModal`?